routes/dms.js

```diff
@@ -34,10 +34,11 @@ module.exports = function () {
   // -----------------------------------------------
 
   router.get('/', async (req, res) => {
+    let posts =  res.locals.posts
```

Comment:
@sagargg what if no posts? Maybe you could do `let posts = res.locals.posts || []`.

Also, note that you have double space after `=` sign.